rust/src/delta.rs

```diff
@@ -1349,32 +1358,51 @@ impl<'a> DeltaTransaction<'a> {
     /// This method will retry the transaction commit based on the value of `max_retry_commit_attempts` set in `DeltaTransactionOptions`.
     pub async fn commit(
         &mut self,
-        operation: Option<DeltaOperation>,
+        operation: DeltaOperation,
```

Comment:
Changing this public method signature will result in backward compatibility issues. Any concerns? Is backward compatibility offered in the APIs?